tests/backup/backup_helper.go

```diff
@@ -2168,25 +2168,6 @@ func IsBackupLocationPresent(bkpLocation string, ctx context.Context, orgID stri
 	return false, nil
 }
 
-// IsCloudCredPresent checks whether the Cloud Cred is present or not
```

Comment:
Why do you want to move this?

Reply:
Need to use it in CreateS3BackupLocation in common.go for checking the cloudcred status.

Reply:
Okay